frontend/lib/config/LocaleConfig/resources.ts

```diff
@@ -1,3 +1,4 @@
+/* eslint-disable @typescript-eslint/no-unsafe-assignment */
```

Comment:
Needs to be removed. Fix typescript error instead of ignoring it.

Reply:
When I remove that, there is some problem with the type any in imports. Do you know how to fix that? Previously it was working fine, but since then is showing this error. All imports is being done the same way than others languages. Can you check it? Thank you.

![image](https://github.com/StanGirard/quivr/assets/37830968/8439c0bf-a923-47b8-8de6-4a2db4cc54e3)